dashboard/app/assets/stylesheets/application.scss

```diff
@@ -962,7 +962,9 @@ $default-modal-width: 640px;
 }
 
 .header_level_container {
-  display: inline-block;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
In moving stage header stuff to flex box, there were a couple scenarios where other stuff in the header didn't line up quite correctly. This fixes that. I don't love that I'm making this unscoped of a change, but feel pretty okay abou tit.